src/components/mobileservices/validator/rules/MaxLengthRule.js

```diff
@@ -0,0 +1,34 @@
+import { get } from 'lodash-es';
+import { ValidationRuleBaseClass } from './ValidationRuleBaseClass';
+
+/**
+ * This is a very simple rule. Checks that the configured field value length is greater than the one that is expected.
```

Comment:
The description here is a bit confusing.
I think it would better 'Checks that the passed in value has a length smaller or equal than the configured maxlenght'